src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -40,7 +41,8 @@ const DomainList = (props) => {
     isCommunity,
     isBlockedSection,
     isDomain,
-    withKarma
+    withKarma,
+    isFromUserFragment
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new prop `isFromUserFragment` is added without any default value. It's a good practice to provide default values for all props to prevent unexpected behavior if the prop is not provided.
```diff
-    isFromUserFragment
+    isFromUserFragment = false
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->